Cargo.toml

```diff
@@ -10,8 +10,8 @@ edition = "2021"
 [workspace.dependencies]
 ttoken-types = { path = "./types", version = "=0.1.0" }
 
-execution-core = { path = "../rusk/execution-core" }
-rusk-abi = { path = "../rusk/rusk-abi", default-features = false }
+dusk-core = { version = "0.1.0", git = "https://github.com/dusk-network/rusk/", branch = "master", default-features = false }
```

Comment:
One point to take into consideration. Should we fix this to the release branch of Rusk instead of master?